bower.json

```diff
@@ -22,14 +22,43 @@
 		"tests"
 	],
 	"dependencies": {
-		"polymer": "^1.2.0",
+		"polymer": "Polymer/polymer#^2.0.0",
```

Comment:
Shouldn't this be polymer `1-2` or am I mixing things up?

Reply:
`"polymer": "Polymer/polymer#1.9 - 2"` seems to be correct version.
Updated it.